modules/oe_starter_content_person/oe_starter_content_person.info.yml

```diff
@@ -0,0 +1,38 @@
+name: OpenEuropa Starter Content Person
+type: module
+description: A person content type.
+core_version_requirement: ^9.2
+package: OpenEuropa Starter Content
+dependencies:
+  - address:address
+  - drupal:media
+  - drupal:node
+  - drupal:text
+  - oe_content:oe_content
```

Comment:
The ct was created while oe_content was enabled, which causes base fields to be added in the form mode export.
An alternative would be to disable oe_content and get rid of the base fields.
TBD.